src/libs/utils/isEventPathInteractive.ts

```diff
@@ -0,0 +1,24 @@
+export const interactiveElementsSelectors: string[] = [
+  'a',
+  'details',
+  'dialog',
+  'menu',
+  'menuitem',
+  'summary',
+  'button',
+  'datalist',
+  'input',
+  'label',
+  'meter',
+  'optgroup',
+  'option',
+  'select',
+  'textarea',
+  '[onclick]'
+];
+const htmlElementInteractive = (element: HTMLElement) =>
```

Comment:
You should keep previous name `isHtmlElementInteractive`. This was also more explicit since this is a function and it returns a boolean.